package.json

```diff
@@ -67,7 +67,7 @@
     "watch": "gulp watch",
     "lint": "balena-lint -e ts -e js --typescript --fix automation/ lib/ typings/ tests/ gulpfile.js",
     "update": "ts-node --transpile-only ./automation/update-module.ts",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build && npx oclif-dev manifest"
```

Comment:
Nice! 👍   If the manifest is generated in the `prepublishonly` script, I think it would only be available for the `npm install` method, not the standalone zip package and the native installers (Windows and macOS). What if `npx oclif-dev manifest` was added to the `build` npm script? And then the `oclif.manifest.json` file would also need to be added to:
* This list of files used when building the native installers: https://github.com/balena-io/balena-cli/blob/v12.1.16/patches/@oclif+dev-cli+1.22.0.dev.patch#L89-L103
* The `pkg.assets` section of `package.json` so that the file is included in the standalone zip packages.